Documentation/teaching/index.rst

```diff
@@ -12,7 +17,7 @@ each topic there are two parts:
   is presented with a starting coding skeleton and with in-depth tips
   on how to solve the exercises
 
-They are based on the labs of the `Operatings Systems 2
+This content is based on the `Operatings Systems 2
 <http://ocw.cs.pub.ro/courses/so2>`_ course, Computer Science
```

Comment:
The Computer Science and Engineering Department, the Faculty of Automatic Control and Computers, University POLITEHNICA of Bucharest.